src/tests/request.test.js

```diff
@@ -263,4 +264,133 @@ describe('/api/v1/requests', () => {
       expect(JSON.parse(res.text).error).to.equal('You\'ve not make any requests');
     });
   });
+
+  describe('POST / ReturnTrip', () => {
```

Comment:
I thought I have merged your return trip task, why is the changes still here?

Reply:
I checkout from the branch while creating this current branch